sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -117,6 +125,36 @@ public void setPrimaryTransports(List<TransportType> primaryTransports){
 		this.requestedPrimaryTransports = primaryTransports;
 	}
 
+	public void setSupportedSecondaryTransports(List<TransportType> secondaryTransports){
+		this.supportedSecondaryTransports = secondaryTransports;
+	}
+
+	public void setSupportedServices(SessionType serviceType, List<Integer> order){
+		if(supportedServicesMap == null){
+			supportedServicesMap = new HashMap<>();
+		}
+		this.supportedServicesMap.put(serviceType, order);
+	}
+
+	public void setRegisteredHighBandwidth(boolean registeredHighBandwidth){

```

Comment:
This method seems to be confusing. Shouldn't we set our registered transports when we receive the control frame ACK?